hack/windows/Install-OVS.ps1

```diff
@@ -5,22 +5,17 @@
   .PARAMETER DownloadURL
   The URL of the OpenvSwitch package to be downloaded.
 
-  .PARAMETER DownloadDir
-  The path of the directory to be used to download OpenvSwitch package. The default path is the working directory.
-
   .PARAMETER OVSInstallDir
   The target installation directory. The default path is "C:\openvswitch".
```

Comment:
I would add the following here to the description:

> If this script is run from a HostProcess Container, this path must be an absolute "host path", and no be under the container's mount path.

This is based on my understanding, so let me know if I am still missing something.

Reply:
Sure, added.